Velaptor/Services/IDotnetService.cs

```diff
@@ -0,0 +1,27 @@
+namespace Velaptor.Services;
```

Comment:
Please add a file header to the top of the file.

> [!Important]
> All new files need to have this added.

```suggestion
// <copyright file="IDotnetService.cs" company="KinsonDigital">
// Copyright (c) KinsonDigital. All rights reserved.
// </copyright>

namespace Velaptor.Services;
```